homeassistant/components/ipp/sensor.py

```diff
@@ -119,6 +120,7 @@ async def async_setup_entry(
                     name=marker.name,
                     icon="mdi:water",
                     native_unit_of_measurement=PERCENTAGE,
+                    state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Just a side note for anyone interested in improving this integration:

This entity description can be broken out to a constant if we set the marker index as an attribute of a new `MarkerIPPSensor` entity class and override the unique_id of that entity.

Using constant entity descriptions is much easier to read.